[ERROR]: during executing openroad script /openlane/scripts/openroad/resizer.tcl
[ERROR]: Log: designs/project2/runs/RUN_2025.05.03_12.34.36/logs/placement/8-resizer.log
[ERROR]: Last 10 lines:
[INFO DPL-0035]  _164_
[INFO DPL-0035]  _164_
[INFO DPL-0035]  input3
[INFO DPL-0035]  _275_
[INFO DPL-0035]  _174_
[INFO DPL-0035]  _174_
[INFO DPL-0035]  _288_
[ERROR DPL-0036] Detailed placement failed.
Error: resizer.tcl, 66 DPL-0036
child process exited abnormally

[ERROR]: Creating issue reproducible...
[ERROR]: Step 8 (placement) failed with error:
-code 1 -level 0 -errorstack {INNER {invokeStk1 throw_error} CALL {run_tcl_script -tool openroad -no_consume /openlane/scripts/openroad/resizer.tcl -indexed_log /openlane/designs/project2/runs/RUN_2025.05.03_12.34.36/logs/placement/8-resizer.log -save to=/openlane/designs/project2/runs/RUN_2025.05.03_12.34.36/tmp/placement,name=resizer,def,odb,sdc,netlist,powered_netlist} CALL {run_openroad_script /openlane/scripts/openroad/resizer.tcl -indexed_log /openlane/designs/project2/runs/RUN_2025.05.03_12.34.36/logs/placement/8-resizer.log -save to=/openlane/designs/project2/runs/RUN_2025.05.03_12.34.36/tmp/placement,name=resizer,def,odb,sdc,netlist,powered_netlist} CALL run_resizer_design CALL run_placement CALL run_placement_step CALL {run_non_interactive_mode -design project2}} -errorcode NONE -errorinfo {
    while executing
"throw_error"
    (procedure "run_tcl_script" line 219)
    invoked from within
"run_tcl_script -tool openroad -no_consume {*}$args"
    (procedure "run_openroad_script" line 2)
    invoked from within
"run_openroad_script $::env(SCRIPTS_DIR)/openroad/resizer.tcl -indexed_log [index_file $::env(placement_logs)/resizer.log] -save "to=$::env(placement_t..."
    (procedure "run_resizer_design" line 8)
    invoked from within
"run_resizer_design"
    (procedure "run_placement" line 21)
    invoked from within
"run_placement"
    (procedure "run_placement_step" line 8)
    invoked from within
"run_placement_step"} -errorline 1
[ERROR]: Failed to create manufacturability and metric reports:
[ERROR]: Traceback (most recent call last):
  File "/openlane/scripts/generate_reports.py", line 184, in <module>
    cli()
  File "/nix/store/8ywrzlq8yyqrif2sfvbgg8vsxhj80ry7-python3-3.11.9-env/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/8ywrzlq8yyqrif2sfvbgg8vsxhj80ry7-python3-3.11.9-env/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/nix/store/8ywrzlq8yyqrif2sfvbgg8vsxhj80ry7-python3-3.11.9-env/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/8ywrzlq8yyqrif2sfvbgg8vsxhj80ry7-python3-3.11.9-env/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/openlane/scripts/generate_reports.py", line 51, in cli
    report = Report(design, tag, design_name, params, run_path).get_report()
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/openlane/scripts/report/report.py", line 809, in get_report
    *self.extract_all_values(),
     ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/openlane/scripts/report/report.py", line 359, in extract_all_values
    power_report = Artifact(
                   ^^^^^^^^^
  File "/openlane/scripts/report/report.py", line 62, in __init__
    self.pathname = matches[0]
                    ~~~~~~~^^^
IndexError: list index out of range
[ERROR]: Flow failed.
